psd-web/spec/features/sign_in_spec.rb

```diff
@@ -5,18 +5,22 @@
 
   let(:investigation) { create(:project) }
   let(:user) { create(:user, :activated, has_viewed_introduction: true) }
+  let(:password) { "2538fhdkvuULE36f" }
 
-  def fill_in_credentials
-    visit "/sign-in"
-
+  def fill_in_credentials(password_override: nil)
```

Comment:
I think ideally we should always pass in both the email and the password to this helper?

```suggestion
  def fill_in_credentials(email:, password:)
```

Reply:
I think we can wait until we need it.